src/Sarif.Viewer.VisualStudio.Core/Resources.resx

```diff
@@ -405,4 +405,7 @@ Do you want to open this file in Visual Studio?</value>
   <data name="TraceLog_SarifFileProcessed" xml:space="preserve">
     <value>The SARIF log file `{0}` processed.</value>
   </data>
+  <data name="WarningResultsHaveBeenFiltered" xml:space="preserve">
+    <value>The SARIF viewer received one or more SARIF logs, but the results may be hidden because of the ErrorList filter settings.</value>
```

Comment:
>The SARIF viewer received one or more SARIF logs, but the results may be hidden because of the ErrorList filter settings. [](http://example.com/codeflow?start=11&length=121)

@EasyRhinoMSFT pls review the phase